configs/authorization.json

```diff
@@ -435,6 +513,16 @@
           "end_user": {}
         }
       },
+      "GetRolesOfUser": {
```

Comment:
"GetRealmRolesForUser" not removed but not present in this file... Was it missing by error ?

Reply:
Should be removed, where I missed it ?